packages/read-emails/frontend/react/src/components/Layout.jsx

```diff
@@ -26,9 +31,11 @@ const Layout = ({ children, showMenu = false, disconnectUser, refresh }) => {
               <span className="hidden-mobile">Refresh</span>
             </button>
             <div className="hidden-mobile">·</div>
-            <button onClick={handleDisconnect}>
+            <button onClick={handleDisconnect} disabled={isDisconnecting}>
               <img src={IconLogout} alt="Logout" height="16" />
-              <span className="hidden-mobile">Disconnect account</span>
+              <span className="hidden-mobile">
+                {isDisconnecting ? 'Disconnect...' : 'Disconnect account'}
```

Comment:
should say "Disconnecting..."

Reply:
Fixed!

Reply:
> <img alt="Screenshot 2022-11-03 at 12 26 41 PM" width="1472" src="https://user-images.githubusercontent.com/43771331/199804509-8065ee28-fb96-484b-9f48-44768d45f406.png">
> 
> is there a way to check for duplicate emails? It's showing me each email from my Outlook inbox twice

For the duplicate case, I think it's because it's grabbing individual messages in the same thread, so they are showing as the same subject. That changes will be in the backend file which is not included in this PR.

Reply:
![Screenshot 2022-11-04 at 9 20 16 AM](https://user-images.githubusercontent.com/43771331/200011402-bcd6de60-5c24-4812-91b9-09cbe95b1421.png)
I think it's showing the email from both the inbox and sent folder (I sent these emails to myself).  We should filter out sent emails (in another ticket) assuming product doesn't want to have them in the sample app's main view (since the distinction between sent and received is not obvious in the current UI